sdk/storage/azure-storage-file-datalake/GEN1_GEN2_MAPPING.md

```diff
@@ -0,0 +1,191 @@
+<h1>Mapping from ADLS Gen1 API -> ADLS Gen2 API</h1>
+<table style="background:white">
+<thead>
+<tr>
+<th>ADLS Gen1 API</th>
+<th>Note for Gen1 API</th>
+<th>ADLS Gen2 API</th>
+<th>Note for API Mapping</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td>access/exists</td>
+<td>To check if file/directory exists.</td>
+<td>N/A</td>
+<td>User can use Gen2 API: <strong>create_file(if_non_match=&#39;*&#39;)<strong> or </strong>create_directory(if_non_match=&#39;*&#39;)</strong> so that the operation will fail on exist.</td>
```

Comment:
just checking if this should be non_match instead of none_match

Reply:
good catch!